src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -466,7 +478,7 @@ class GroupReadsByUmi
 
     logger.info(f"Accepted $kept%,d reads for grouping.")
     if (filteredNonPf > 0) logger.info(f"Filtered out $filteredNonPf%,d non-PF reads.")
-    logger.info(f"Filtered out $filteredPoorAlignment%,d reads that were not part of a high confidence mapped read pair.")
+    logger.info(f"Filtered out $filteredPoorAlignment%,d reads due to low mapping quality.")
```

Comment:
Well not exactly true, since we could have a really high mapq but the mates are mapped to different chromosomes.